### PR TITLE
223 create animation component

### PR DESCRIPTION
**Pull request by osvegn**
## New features
- Create AnimationComponent.
- Create tests for this component.

### CODE REVIEW

**Review by LucasTesnier — APPROVED** [review]
NTR!

File: tests/unit_tests/libs_tests/SFML_tests/Components/AnimationComponent.cpp [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+** File description:
+** AnimationComponent
+*/
+
+#include <criterion/criterion.h>
+#include "AnimationComponent.hpp"
+
+Test(AnimationComponent, create_class)
+{
+    AnimationComponent a;
```
Comment (ZQUEMA) [inline tests/unit_tests/libs_tests/SFML_tests/Components/AnimationComponent.cpp:13]: Precise that the component are in the namespace `ecs`

**Review by ZQUEMA — CHANGES_REQUESTED** [review]
One minor change is required

**Review by ZQUEMA — DISMISSED** [review]

File: tests/unit_tests/libs_tests/SFML_tests/Components/AnimationComponent_tests.cpp [hunk 2]
```diff
@@ -0,0 +1,16 @@
+/*
+** EPITECH PROJECT, 2022
+** R-Type [WSL : Ubuntu]
+** File description:
+** AnimationComponent
+*/
+
+#include <criterion/criterion.h>
+#include "AnimationComponent.hpp"
```
Comment (ZQUEMA) [inline tests/unit_tests/libs_tests/SFML_tests/Components/AnimationComponent_tests.cpp:9]: Is it the correct path ? 
Should not it be `GraphicECS/SFML/Components/...` ?

**Review by 31Nathan — APPROVED** [review]
LGTM

File: tests/unit_tests/libs_tests/SFML_tests/Components/AnimationComponent_tests.cpp (hunk 2 above)
Comment (osvegn) [reply tests/unit_tests/libs_tests/SFML_tests/Components/AnimationComponent_tests.cpp:9]: It works

**Pull request merged**